packages/lodestar/src/chain/chain.ts

```diff
@@ -333,6 +333,7 @@ export class BeaconChain extends (EventEmitter as { new(): ChainEventEmitter })
           this.db.chain.setFinalizedBlockRoot(finalizedBlockRoot),
         ]);
         this.emit("finalizedCheckpoint", newState.finalizedCheckpoint);
+        this.emit("newEpoch", computeEpochAtSlot(this.config, finalizedBlock.message.slot), this.blockProcessingQueue);
```

Comment:
This won't work if chain stops finalizing. I would suggest waiting for clock in https://github.com/ChainSafe/lodestar/pull/648/files#diff-7602fa529a155c161f8bf81ea5328d5bR6
to be merged as it will provide epoch listener